src/Builder/FrontendDialectTransformer.cpp

```diff
@@ -324,7 +334,8 @@ class FrontendGenImpl {
       onnx::TypeProto elem_type = input_seq_type.elem_type();
       assert(elem_type.value_case() == onnx::TypeProto::kTensorType &&
              "expect tensor inside sequence type");
-      Type mlir_elem_type = ImportTensorType(elem_type);
+      std::string s;
```

Comment:
a little bit easier to read if you rename `s` to `dimParam`

or consider adding convenience methods `ImportTensorType` and `ImportType` without the dimParam parameter, or make dimParam a pointer that defaults to nullptr

Reply:
Done. Changed to use optional arguments.